templates/UmbracoExtension/Client/public/umbraco-package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../../umbraco-package-schema.json",
+  "id": "UmbracoExtension",
+  "name": "UmbracoExtension",
+  "version": "0.0.0",
+  "allowPackageTelemetry": false,
+  "extensions": [
+    {
+      "name": "UmbracoExtension EntryPoint",
+      "alias": "PROJECT_NAME_KEBABCASE_FOR_NPM.entrypoint",
+      "type": "backofficeEntryPoint",
```

Comment:
Looks like both would work, but the example you have in your entry point file looks more like the `bundle` type than the `backofficeEntryPoint` type. Maybe someone on the Backoffice team has thoughts on this.

Reply:
AFAIK Bundles is for registering manifests only, where an entrypoint you may do other logic inside it.
Such as to do the AUTH stuff with the C# OpenAPI spec or perhaps unregister/exclude extensions etc...

But lets wait for someone from the frontend core crew to chime in.

Reply:
Hi @warrenbuckley 
I would say use the EntryPoint for what the entry point is needed for, but do not use it to registere more manifests. For that its better to use the Bundle.
Main reason begin a bundle knows the extensions it registres. Meaning if later and that could be milliseconds later, it knows how to unregisters it self. Useful if someone decides to unregister it.

So my opinion would be, in order to be the most correct as you can, then make both, one bundle and one entry point so people can see how the can execute code up front and use the bundle to lead the way for more extensions. :-)

Reply:
Seems overkill don't you think to have the following and is more concepts to teach/get across unless this really is **THE WAY**

`umbraco-package.json -> backofficeEntryPoint -> bundle`

Reply:
Just to confirm you got everything right in your above comment. :-)

But I would still suggest this style:
umbraco-package.json -> bundle -> backofficeEntryPoint

Which would result in anything added more begin added should go to the bundle:
umbraco-package.json -> bundle -> dashboard

In this way avoiding to use the entry-point to do registration of other extension-manifest. Meaning the entry-point is purely needed for the act of running some JS code. Then if people then dont need to run code, they can ignore that specific type.

I think that is the most correct style.

Reply:
OK I will rework this, thanks :)